plugins/confluence.go

```diff
@@ -46,12 +48,26 @@ func (p *ConfluencePlugin) GetAuthorizationHeader() string {
 	return lib.CreateBasicAuthCredentials(p)
 }
 
+func validateURL(urlStr string) bool {
+	parsedURL, err := url.Parse(urlStr)
+	if err != nil {
+		return false
+	}
+
+	if parsedURL.Scheme != "https" {
+		return false
+	}
+
+	return true
+}
+
+
 func (p *ConfluencePlugin) DefineCommand(channels Channels) (*cobra.Command, error) {
 	var confluenceCmd = &cobra.Command{
 		Use:   fmt.Sprintf("%s <URL>", p.GetName()),
 		Short: "Scan Confluence server",
 		Long:  "Scan Confluence server for sensitive information",
-		Args:  cobra.ExactArgs(1), // This makes sure only one argument (URL) is provided.
+		Args:  cobra.ExactArgs(1), 
```

Comment:
See [this line](https://github.com/itay-goldraich/2ms/blob/7babb1704350af61302cc7aab79aa24e7173e599/plugins/git.go#L39) in `git.go`: `Args:  cobra.MatchAll(cobra.ExactArgs(1), validGitRepoArgs)`.

Instead of validating the URL on `Run`, where you need to send the error into the `Errors` channel and create an error message, you can add a custom validator to `cobra` and it will fail with the `help` message and your error.

Reply:
Thanks for the feedback, I'll take a look.